LICENSE

```diff
@@ -1,6 +1,7 @@
-MIT License
+The MIT License (MIT)
```

Comment:
Just make sure that the GitHub API still registers this license as the MIT License after you change stuff like this (the license text is seeded by GitHub)